### DISCUSSION TITLE
LookAt blendshapes

### DISCUSSION THREAD

**Discussion by James-messant** [pixiv/three-vrm#1303] [Help]
Is it possible to control eye movement through blendshapes using this arkit model:
https://github.com/hinzka/52blendshapes-for-VRoid-face?

currentVrm.expressionManager.setValue("EyeLookOutLeft", 0.6);
This line does not work for me even though the model contains an expressionVrm blendshape for it.

I've tried doing : lookAtTarget.position.y = eyeLookUpLeft - eyeLookDownLeft
Which works somewhat okay but was wondering if there was a better way of doing it.

**Comment by ke456-png** [comment]
There are two types of LookAt in VRM: models that use Expression(BlendShape) and models that use Bone.
https://github.com/vrm-c/vrm-specification/blob/master/specification/VRMC_vrm-1.0/lookAt.md#lookattype

If you want to support both, use [VRM.LookAt](https://pixiv.github.io/three-vrm/packages/three-vrm/docs/classes/VRMLookAt.html).

example: https://github.com/pixiv/three-vrm/blob/dev/packages/three-vrm/examples/lookat.html
